Fenced_Frames_Ads_Reporting.md

```diff
@@ -218,7 +218,9 @@ window.fence.setReportEventDataForAutomaticBeacons({
 
 If `setReportEventDataForAutomaticBeacons` is invoked, the browser will send an automatic beacon to all URLs registered via registerAdBeacon for the given event, but it will only send an event data body (the information in eventData) with the HTTP request to destinations specified in the destination field. This means that invoking setReportEventDataForAutomaticBeacons acts as an opt-in by the fenced frame document to allow sending the beacon to all registered URLs, aligning with cross-origin security principles.
 
-If `setReportEventDataForAutomaticBeacons` is not invoked, the browser will not send an automatic beacon to any registered URLs.
```

Comment:
Might this be more clearly presented with a table?

Reply:
Also, while I know that this doc is primary geared towards fenced frames, it would be useful IMHO to clarify how this impacts PAAPI running via IFRAMEs as well.

Reply:
Updated to be a table.

Iframes created using Protected Audience have access to the same reporting features as fenced frames. I've updated the "Objective" section to explicitly mention that.